Point2D p0
Point2D p1
uint32 color
